### ISSUE TITLE
Add a "Site Category" field to site info

### ISSUE THREAD

**Issue by docwilmot** [backdrop/backdrop-issues#1543]
Was thinking about how to improve Project Browser, and thought it would be nice to have a "recommended downloads" panel. But how would PB know what to suggest? Let's allow a user to store info about the site at `admin/config/system/site-information`. Maybe a main field for "Club site" or "church site" or "NGO" etc, then a tag field for subcategories. 
If we add that field to the installer, we could even have recommendations for downloads show up right away on the front page, instead of the add content text.

**Comment by klonos** [comment]
Not a bad idea. We could also have the most downloaded/used contrib as recommended downloads.

**Comment by docwilmot** [comment]
@jenlampton what do you think of this? I'm on an issue cleanup spree here, so feel free to close this if its not a good idea.

**Comment by jenlampton** [comment]
And we'd also need to add it to .info files for modules to pull in the data?  How do you see this working?

**Comment by docwilmot** [comment]
I saw it working with two of my other pet suggestions:
[Replace "packages" in project info files with "Tags"](https://github.com/backdrop/backdrop-issues/issues/37)
[Features/Packager project types](https://github.com/backdrop/backdrop-issues/issues/1238#issuecomment-142062639)

Imagine you choose "Club site" when youre installing, and then when youre done Installer module searches for modules tagged as "club", and pops up a suggested module called "Club package", which if installed loads all the config, content types, theme, layouts to build one of those. Instant sites!

But yes, premature suggestion without categorization info in project info files.

**Comment by klonos** [comment]
...or instead of choosing through a predefined list of site categories (that we'd perhaps have to update every now and then), we could ask the users to freely describe their website type, get the text they enter, strip it of common words like "and", "the" "of", "a" etc. and then do a search with what's left for any matching tags tags in all projects. The more tags a project has that match with the search query, then higher it would bobble up the "Recommended downloads" list.

**Comment by klonos** [comment]
...but yes, #37 needs to happen first.

**Comment by jenlampton** [comment]
I'm thinking this is something we should implement on backdropcms.org first. It will give us a nice safe place to test and refine until we get it right :)
